src/rkopenmdao/butcher_tableau.py

```diff
@@ -61,16 +90,32 @@ def p(self):
     def p(self, p):
         self._p = p
 
+    def minPorder(self):
```

Comment:
Function names should be in snake case, i.e. `min_p_order` in this case. Have a look at [pylint](https://pylint.pycqa.org/en/latest/index.html), it can help you with such things.